src/main/java/net/minestom/server/item/ItemMetaBuilder.java

```diff
@@ -146,6 +149,30 @@
         return this;
     }
 
+    @Contract("_ -> this")
+    public @NotNull ItemMetaBuilder canPlaceOn(@NotNull Set<@NotNull Block> blocks) {
```

Comment:
I think it might be nice to have varargs methods for these setters that just overload the set methods.

Reply:
Done :)